.github/workflows/book.yml

```diff
@@ -0,0 +1,38 @@
+name: deploy-book
+
+# Only run this when the master branch changes
+on:
+  push:
+    paths:
+      - 'docs/**'
+    branches:
+      - 'master'
+
+# This job installs dependencies, build the book, and pushes it to `gh-pages`
+jobs:
```

Comment:
i.e. the book get's rebuilt and deployed after a PR? Neat...

Reply:
yea!